virtcontainers/fc.go

```diff
@@ -395,7 +395,7 @@ func (fc *firecracker) createDiskPool() error {
 	defer span.Finish()
 
 	for i := 0; i < fcDiskPoolSize; i++ {
-		driveID := "drive-" + strconv.Itoa(i)
+		driveID := "drive_" + strconv.Itoa(i)
```

Comment:
A tiny change I know, but if we think this could change again, maybe a `func fcDriveIndexToID(i int) string` might be a good idea?

Reply:
done